src/templates/next/components/internal/ContentPageHeader/ContentPageHeader.tsx

```diff
@@ -7,21 +7,29 @@ import { Paragraph } from "../../../typography/Paragraph"
 const ContentPageHeader = ({
   title,
   summary,
+  lastUpdated,
   breadcrumb,
   buttonLabel,
   buttonUrl,
   LinkComponent,
 }: ContentPageHeaderProps) => {
   return (
-    <div className="bg-site-primary-100 px-4 py-8 lg:px-5 lg:py-16">
+    <div className="bg-site-primary-100 px-4 py-8 md:py-10 lg:px-5 lg:py-16">
       <div className="max-w-[848px] flex flex-col gap-8 lg:gap-12 mx-auto">
         <div className="hidden lg:block">
           <Breadcrumb links={breadcrumb.links} LinkComponent={LinkComponent} />
         </div>
-        <div className="flex flex-col gap-4 lg:gap-8">
+        <div className="flex flex-col">
           <h1 className="text-[2.75rem] leading-tight lg:text-[3.75rem] font-semibold text-content-strong">
             {title}
           </h1>
+          <div className="pt-6 lg:pb-2">{`Last updated ${new Date(
+            lastUpdated,
+          ).toLocaleDateString(undefined, {
+            day: "2-digit",
+            month: "short",
+            year: "numeric",
+          })}`}</div>
```

Comment:
nit: I think there might be some value in us standardising the date format throughout the entire site. Can I trouble you to extract out this as a util function and update the footer + article layout as well?

Reply:
Refactored in https://github.com/isomerpages/isomer-components/pull/88/commits/790058b20a8b97ef7f0b147e159f0d125d65b5ea!